litewallet/Platform/Extensions.swift

```diff
@@ -558,3 +558,17 @@ extension Dictionary where Key: ExpressibleByStringLiteral, Value: Any {
 		return jstring
 	}
 }
+
+extension UInt64 {
```

Comment:
Consider Extending a Protocol (Optional): If similar safe arithmetic methods are needed for other numeric types (e.g., Int, Double), you could introduce a protocol for safe arithmetic. This promotes code reuse and reduces duplication.

protocol SafeArithmetic {
    func safeAddition(_ value: Self) -> Self?
    func safeSubtraction(_ value: Self) -> Self?
}